bcml/__main__.py

```diff
@@ -55,9 +55,9 @@ def configure_cef(debug):
     if not cache.exists():
         cache.mkdir(parents=True, exist_ok=True)
 
-
+        

```

Comment:
Most important change ever. ;)

That being said, please get rid of it. Unnecessary whitespace edits make diffing across versions really annoying.